src/libs/ReportUtils.js

```diff
@@ -1587,32 +1587,41 @@ function getReportIDFromLink(url) {
     return reportID;
 }
 
+/**
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function canRequestMoney(report) {
+    return (!isPolicyExpenseChat(report) || report.isOwnPolicyExpenseChat);
+}
+
 /**
  * @param {Object} report
  * @param {Array} reportParticipants
  * @param {Array} betas
  * @returns {Array}
  */
-function getIOUOptions(report, reportParticipants, betas) {
+function getMoneyRequestOptions(report, reportParticipants, betas) {
```

Comment:
Is this for Requests only? Or sending payments as well?

Reply:
They are for both. We are using the moneyRequest terminology for request money, split bill and send money options

Reply:
Ah yes! I caught up on the design doc section linked in the issue